src/main/kotlin/net/atos/zac/admin/ZaakafhandelParameterBeheerService.kt

```diff
@@ -254,10 +258,17 @@ class ZaakafhandelParameterBeheerService @Inject constructor(
     }.toSet().let(nieuweZaakafhandelParameters::setMailtemplateKoppelingen)
 
     private fun mapVorigResultaattypeOpNieuwResultaattype(
+        vorigResultaattypeUUID: UUID,
         nieuweResultaattypen: List<ResultaatType>,
-        vorigResultaattypeUUID: UUID
     ): UUID? =
         ztcClientService.readResultaattype(vorigResultaattypeUUID).let { resultaattype ->
             nieuweResultaattypen.firstOrNull { it.omschrijving == resultaattype.omschrijving }?.url?.extractUuid()
         }
+
+    private fun mapSmartDocuments(
+        vorigeZaakafhandelparametersUUID: UUID,
```

Comment:
Better to use EN as much as possible where we can?
```suggestion
        previousZaakafhandelparametersUUID: UUID,
```